arrangor/src/main/kotlin/no/nav/amt/tiltak/arrangor/ArrangorServiceImpl.kt

```diff
@@ -1,37 +1,17 @@
 package no.nav.amt.tiltak.arrangor
 
 import no.nav.amt.tiltak.core.domain.arrangor.Arrangor
-import no.nav.amt.tiltak.core.domain.arrangor.ArrangorUpdate
 import no.nav.amt.tiltak.core.port.ArrangorService
-import no.nav.amt.tiltak.data_publisher.DataPublisherService
-import no.nav.amt.tiltak.data_publisher.model.DataPublishType
 import no.nav.amt.tiltak.tilgangskontroll_tiltaksarrangor.arrangor.AmtArrangorService
-import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
 import java.util.UUID
 
 @Service
 class ArrangorServiceImpl(
 	private val amtArrangorService: AmtArrangorService,
 	private val arrangorRepository: ArrangorRepository,
-	private val dataPublisherService: DataPublisherService
 ) : ArrangorService {
 
-	private val log = LoggerFactory.getLogger(javaClass)
-
-	override fun upsertArrangor(virksomhetsnummer: String): Arrangor {
-		val arrangor = amtArrangorService.getArrangor(virksomhetsnummer) ?: throw RuntimeException("Kunne ikke hente arrangør med orgnummer $virksomhetsnummer")
-
-		return arrangorRepository.upsert(
-			id = arrangor.id,
-			navn = arrangor.navn,
-			organisasjonsnummer = arrangor.organisasjonsnummer,
-			overordnetEnhetNavn = arrangor.overordnetEnhetNavn,
-			overordnetEnhetOrganisasjonsnummer = arrangor.overordnetEnhetOrganisasjonsnummer,
-		).toArrangor()
-			.also { dataPublisherService.publish(it.id, DataPublishType.ARRANGOR) }
```

Comment:
Tanken med å sette source på kafkameldingen er at både amt-tiltak og amt-arrangør kan publisere til topicen i en overgangsperiode. Når du fjerner publiseringen fra amt-tiltak kan du la være å ha med source, men da må du i praksis ta ned begge appene i prod for å være sikker på at du får med alle arrangører. Jeg tror med andre ord det er for tidlig å fjerne denne :)

Reply:
Hmm, det går kanskje greit hvis du starter publisering fra amt-arrangør først og setter source i meldingene derfra til å være amt-arrangør, og endrer sånn at du ignorerer alle meldingene som har source==null i ingesten i amt-tiltak?